src/inputs/Table/Table.js

```diff
@@ -190,31 +252,78 @@ export const Table = (props) => {
         isLoading={isLoading}
         onImport={onImport}
         onExport={onExport}
+        onAddRow={onAddRow ? addRows : null}
+        onDeleteRow={onDeleteRow ? deleteRows : null}
         editMode={editMode}
         customToolbarActions={customToolbarActions}
         labels={toolbarLabels}
+        getSelectedRowsCount={getSelectedRowsCount}
       />
     );
   }, [
+    labels.import,
+    labels.export,
+    labels.fullscreen,
+    labels.addRow,
+    labels.deleteRows,
+    isFullscreen,
+    toggleFullscreen,
+    isReady,
+    isLoading,
+    onImport,
+    onExport,
+    onAddRow,
+    addRows,
+    onDeleteRow,
+    deleteRows,
+    editMode,
     customToolbarActions,
+    getSelectedRowsCount,
+  ]);
+
+  const emptyTablePlaceholder = useMemo(() => {
+    return (
+      <div className={classes.emptyTablePlaceholderDiv} data-cy="empty-table-placeholder">
+        <Typography variant="h5" className={classes.emptyTablePlaceholderTitle}>
+          {labels.placeholderTitle}
+        </Typography>
+        <Typography variant="body1" className={classes.emptyTablePlaceholderBody}>
+          {labels.placeholderBody}
+        </Typography>
+        {onImport ? (
+          <Button
+            key="import-file-button"
+            disabled={!(editMode && !isLoading)}
```

Comment:
This condition can be simplified to be easier to read: `!editMode || isLoading`

Reply:
Please check this again :smile:

Reply:
Done...